Fakebook.Posts/Fakebook.Posts.Domain/Models/Post.cs

```diff
@@ -1,8 +1,22 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Text;
 
-namespace Fakebook.Posts.Domain {
-    public class Post {
+namespace Fakebook.Posts.Domain.Models
+{
+    public class Post
+    {
+        public int Id { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
+        public string Content { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
+        public string Picture { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
+        public string UserEmail { get => throw new NotImplementedException(); private set => throw new NotImplementedException(); }
+        public DateTime CreatedAt { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
+        public List<Comment> Comments { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
+
+        public Post(string userEmail) {
+            if (string.IsNullOrWhiteSpace(userEmail)) {
+                throw new ArgumentException("Email cannot be null or whitespace.", nameof(userEmail));
```

Comment:
again... we're doing TDD. you don't add implementation here, only tests. the constructor should throw an exception.
(now that i think of it, this means the properties don't need to throw, because the tests would never get that far. but it's fine.)

shouldn't the ctor take content as well?

Reply:
don't re-request review when you haven't addressed the previous review!